xknx/io/gateway_scanner.py

```diff
@@ -141,7 +141,7 @@ def __init__(
         tunnelling: bool | None = None,
         tunnelling_tcp: bool | None = None,
         routing: bool | None = None,
-        secure: bool | None = None,
+        secure: bool | None = False,
```

Comment:
Can we remove None here?

Reply:
The idea was if we had secure credentials, but there is a non-secure interface installed as well - we could use both (eg. the secure one is saturated).
"Automatic" doesn't support secure for now - this will be a follow-up